SwipeListExample/examples/actions.js

```diff
@@ -0,0 +1,283 @@
+import React, { useState } from 'react';
+import {
+    StyleSheet,
+    Text,
+    Image,
+    Animated,
+    TouchableOpacity,
+    TouchableHighlight,
+    View,
+} from 'react-native';
+
+import SwipeListView from '../SwipeListView';
+
+export default function Actions() {
+    const [listData, setListData] = useState(
+        Array(20)
+            .fill('')
+            .map((_, i) => ({
+                key: `${i}`,
+                text: `item #${i}`,
+                initialLeftActionState: i % 2,
```

Comment:
I think this one should be a bool? Maybe you mean to write `i % 2 !== 0`?

Reply:
Yep, i think so. I will fix it.

Reply:
Fixed